src/artemis/experiment_plans/tests/test_optimise_attenuation_plan.py

```diff
@@ -48,6 +57,7 @@ def get_good_status():
     return status
 
 
+@pytest.mark.skip(reason="Flakey test which is refactored in another PR")
```

Comment:
Skipped this test for now as it's refactored in https://github.com/DiamondLightSource/python-artemis/pull/763 (although still isn't working properly after that)

Reply:
Coverage is low due to this test being skipped, but #763 should fix that